AWS.md

```diff
@@ -1,7 +1,18 @@
 General
 -------
 
- * Use CloudFormation or ElasticBeanStalk to define infrastructure in AWS.
+ * Use [CDK](https://github.com/guardian/cdk) CloudFormation or ElasticBeanStalk to define infrastructure in AWS
+   * Prefer to [import resources](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/resource-import.html) into CFN over manually configuring them
+ * Tag resources with:
+   * `Stack` - the broad umbrella the service sits under e.g. `deployTools`
+   * `Stage` - the environment, typical values are:
+     * `PROD` for production
+     * `CODE` for pre-production
+     * `INFRA` for infrastructure or singleton resources e.g. [elasticsearch-node-rotation](https://github.com/guardian/elasticsearch-node-rotation)
+   * `App` - the individual service e.g. `prism`
+ * If a resource needs to be shared across multiple stacks, prefer to define it in it's own CFN template with stage `INFRA`
+ * Prefer continuous delivery of infrastructure via Riff-Raff over manual deployment
```

Comment:
It could be worth explaining the rationale (e.g. better audit trail and safety mechanisms in Riff-Raff) for this?

Reply:
👍🏽  Done.